### PR TITLE
Updated quickstart to reflect newest zenml version

### PR DESCRIPTION
**Pull request by alexej-zenml**
## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Describe changes
Update to the quickstart notebook

### CODE REVIEW

File: examples/quickstart/quickstart.ipynb [hunk 1]
```diff
@@ -166,7 +167,7 @@
     "id": "QZI9i2FJ0k3H"
    },
    "source": [
-    "The first step is an `import` step that downloads the MNIST dataset and samples the first hundred rows for demo purposes."
+    "The first step is an `import` step that downloads the MNIST dataset and returns 4 numpy arrays as its output. "
```
Comment (alex-zenml) [inline examples/quickstart/quickstart.ipynb:170]:
```suggestion
    "The first step is an `import` step that downloads the MNIST dataset and returns four numpy arrays as its output. "
```

File: examples/quickstart/quickstart.ipynb [hunk 2]
```diff
@@ -196,7 +197,7 @@
     "id": "ma53mucU0yF3"
    },
    "source": [
-    "We then add a `Trainer` step, that takes the normalized data and trains a Keras classifier on the data. Note that the `Output[ModelArtifact]` type helps in writing the model out to our artifact store. "
+    "We then add a `Trainer` step, that takes the normalized data and trains a Keras classifier on the data. Note that the model is not explicitely saved within the step. Under the hood ZenML uses Materializers to automatically persist the Artifacts that result from each step into the Artifact Store."
```
Comment (alex-zenml) [inline examples/quickstart/quickstart.ipynb:200]:
```suggestion
    "We then add a `Trainer` step, that takes the normalized data and trains a Keras classifier on the data. Note that the model is not explicitly saved within the step. Under the hood ZenML uses Materializers to automatically persist the Artifacts that result from each step into the Artifact Store."
```

File: examples/quickstart/quickstart.ipynb [hunk 3]
```diff
@@ -424,12 +429,10 @@
...
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "from zenml.core.repo import Repository"
+    "We did mention above that the Materializer takes care of persisiting your artifacts for you. But how do you acess your runs and their associated artifacts from code? Let's do that step by step."
```
Comment (alex-zenml) [inline examples/quickstart/quickstart.ipynb:435]:
```suggestion
    "We did mention above that the Materializer takes care of persisting your artifacts for you. But how do you access your runs and their associated artifacts from code? Let's do that step by step."
```

File: examples/quickstart/quickstart.ipynb [hunk 4]
```diff
@@ -439,12 +442,21 @@
     "## Get repo"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First off, we load your repository, this is where all your pipelines live. "
```
Comment (alex-zenml) [inline examples/quickstart/quickstart.ipynb:449]:
```suggestion
    "First off, we load your repository: this is where all your pipelines live. "
```

File: examples/quickstart/quickstart.ipynb [hunk 5]
```diff
@@ -510,14 +532,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "fashion_mnist_run = runs[1]"
+    "# Lets first extract out first run on the standard mnist dataset\n",
```
Comment (alex-zenml) [inline examples/quickstart/quickstart.ipynb:535]:
```suggestion
    "# Let's first extract out the first run on the standard mnist dataset\n",
```

**Review by alex-zenml — CHANGES_REQUESTED** [review]

File: examples/quickstart/quickstart.ipynb (hunk 2 above)
Comment (htahir1) [reply examples/quickstart/quickstart.ipynb:200]: Should we then link to materializers in the docs core concepts here?

File: examples/quickstart/quickstart.ipynb [hunk 6]
```diff
@@ -63,7 +63,8 @@
    "outputs": [],
    "source": [
     "# Install the ZenML CLI tool and Tensorflow\n",
-    "!pip install zenml tensorflow"
+    "!pip install zenml \n",
+    "!zenml integration install tensorflow"
```
Comment (htahir1) [inline examples/quickstart/quickstart.ipynb:67]: I think tensorflow is already installed in colabs at least but its fine to keep it in

**Review by htahir1 — COMMENTED** [review]
Some suggestions but overall really nice update!

File: examples/quickstart/quickstart.ipynb (hunk 6 above)
Comment (alexej-zenml) [reply examples/quickstart/quickstart.ipynb:67]: I think some users will run this in a jupyter notebook within their own environments.

Comment (htahir1) [reply examples/quickstart/quickstart.ipynb:67]: Yeah (Y)

**Pull request merged**